spec/models/elastic_indexed_document_spec.rb

```diff
@@ -234,30 +234,18 @@
           expect(ElasticIndexedDocument.search_for(q: 'Brad Miller -yellowstone', affiliate_id: affiliate.id, language: affiliate.indexing_locale).total).to eq(1)
         end
 
-        context 'when query contains specific field queries using "field:" followed by query term or grouped phrase or quoted string' do
```

Comment:
I deleted this block of specs, because we are removing support for searches by a particular field of the document. Dawn confirmed that is is not a feature used by searchers, and re-implementing it would require extra work.

Reply:
I support removing these specs at this time. But to clarify, customers have asked for advanced search functionality o/b/o searchers, including field-specific searching, but boolean operators is a much more frequent request. It remains to be seen how often people are trying to use these kinds of operators. we'll want to think holistically about how we want that future advanced search to work, and then have the specs for it reflect that new reality.